synapse/lib/types.py

```diff
@@ -269,14 +269,12 @@ def norm(self, valu):
 
         return func(valu)
 
-    def repr(self, norm, defval=None):
+    def repr(self, norm):
         '''
-        Return a printable representation for the value.  For types which need no
-        display processing, the defval should be returned.
-
+        Return a printable representation for the value.
         This may return a string or a tuple of values for display purposes.
         '''
-        return defval
+        return str(norm)
```

Comment:
The previous discussion we had about this change was to make this raise s_exc.NoSuchImpl since str(norm) may not be correct.

Reply:
In this case, str(norm) is a decision made specifically by a type implementer ( unlike the "what to do about repr for a type you dont have" case ) so it saves us lines of code since many of the prims would just return str(norm)